en/developer_guide/install_with_vagrant_for_contributors.rst

```diff
@@ -0,0 +1,211 @@
+  
+  .. code-block:: bash
+
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra ./open-orchestra -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-front-demo ./open-orchestra-front-demo -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-media-demo ./open-orchestra-media-demo -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+
+  Clone the stable provisioning repository. Don’t forget to specify the last stable version branch with the ``--branch`` option.
+
+  .. code-block:: bash
+
+    $ git clone git@github.com:open-orchestra/open-orchestra-provision.git --branch=1.0
+
+Override the dns redirection
+----------------------------
+
+In the ``/etc/hosts`` file of your computer add the following lines :
+
+.. code-block:: text
+
+    192.168.33.10   admin.openorchestra.dev
+    192.168.33.10   demo.openorchestra.dev
+    192.168.33.10   media.openorchestra.dev
+
+The two installs use the same IP addresses, so you can’t launch both vagrant at the same time.
+
+
+
+Now follow the next steps for both install versions :
+
+* Install roles from ansible-galaxy
+* Launch the box
+* Install the assets
+* Load the fixtures
+
+
+Install roles from ansible-galaxy
+---------------------------------
+
+Go into ``open-orchestra-provisioning`` directory and install roles needed to launch the box:
+
+.. code-block:: bash
+
+    $ ansible-galaxy install --role-file=galaxy.yml
+
+Launch the box
+--------------
+
+In the ``open-orchestra`` directory, when you launch the box, it will take some time to :
+
+* Import the base box
+* Launch it
+* Run all the provisioning scripts
+
+.. code-block:: bash
+
+    $ vagrant up
+
+Install the assets
+------------------
+
+We are using npm to manage some server side javascript libraries and bower to manage the client side libraries
+
+Connect to the vagrant box using ``vagrant ssh``
+
+Finalise the ``composer`` installation in each project
+
+.. code-block:: bash
+
+    $ cd /var/www/openorchestra && composer run-script post-install-cmd
+    $ cd /var/www/front-openorchestra && composer run-script post-install-cmd
+    $ cd /var/www/media-openorchestra && composer run-script post-install-cmd
+
+Then go in the Back Office project directory inside the box
+
+.. code-block:: bash
+
+    $ cd /var/www/openorchestra
+
+Install the npm dependencies
+
+.. code-block:: bash
+
+    $ npm install
+
+The npm should have also installed the bower component.
+
+Launch the grunt command to generate all assets
+
+.. code-block:: bash
+
+    $ ./node_modules/.bin/grunt
+
+Load the fixtures
+-----------------
+
+In the symfony project directory ``/var/www/openorchestra`` you can load the fixtures provided :
+
+.. code-block:: bash
+
+    $ php app/console doctrine:mongo:fixture:load --env=dev
```

Comment:
This line result to a choice:
`Choose type in all, production, functional (all) :`
It could be a good idea to add a short description to explain what are and when should the contributor load the production fixtures and the functional fixtures?